include/mgp.py

```diff
@@ -1281,21 +1281,39 @@ def is_valid(self) -> bool:
         return self._message.is_valid()
 
     def payload(self) -> bytes:
+        """
+        Supported stream sources:
+          - Kafka
+          - Pulsar
+        """
```

Comment:
I like this! I think we should also add these comments to the C API, shouldn't we?

Reply:
I thought I did but it's not included for some reason...